src/object.d

```diff
@@ -1974,6 +1975,16 @@ void clear(T : Value[Key], Value, Key)(T* aa)
     _aaClear(*cast(void **) aa);
 }
 
+void reserve(T : Value[Key], Value, Key)(ref T aa, size_t ndim)
+{
+    _aaReserve(cast(void **)&aa, typeid(Value[Key]), ndim);
+}
+
+void reserve(T : Value[Key], Value, Key)(T* aa, size_t ndim)
+{
+    _aaReserve(cast(void **)aa, typeid(Value[Key]), ndim);
```

Comment:
Let's rather use simple forwarding instead of the ugly cast twice. That way we only need to maintain the tricky code once:

```d
reserve(*aa, ndim);
```